### PR TITLE
Add config option for using forwarders

### PR DESCRIPTION
**Pull request by essamhassan**
This option will be used by txmgr to allow nops to forward requests through forwarder contracts
This option should be in docs and UI once underlying functionality is ready

### CODE REVIEW

File: core/config/envvar/schema.go [hunk 1]
```diff
@@ -180,6 +180,7 @@ type ConfigSchema struct {
 	EvmMaxInFlightTransactions uint32 `env:"ETH_MAX_IN_FLIGHT_TRANSACTIONS"`
 	EvmMaxQueuedTransactions   uint64 `env:"ETH_MAX_QUEUED_TRANSACTIONS"`
 	EvmNonceAutoSync           bool   `env:"ETH_NONCE_AUTO_SYNC"`
+	EvmUseForwarders           bool   `env:"ETH_USE_FORWARDERS" default:"false"`
```
[the remark this answers is not in the document]
Comment (pinebit) [reply core/config/envvar/schema.go:183]: New env variables must be reflected in the changelog.

**Comment by jmank88** [comment]
Should this overall functionality be behind a global `Feature*` switch as well? Or would that just be redundant?

File: core/chains/evm/config/config.go [hunk 2]
```diff
@@ -866,6 +867,23 @@ func (c *chainScopedConfig) EvmNonceAutoSync() bool {
 	return c.defaultSet.nonceAutoSync
 }
 
+// EvmUseForwarders enables/disables running the NonceSyncer on application start
```
Comment (jmank88) [inline core/chains/evm/config/config.go:870]: Update comment?

File: core/config/envvar/schema.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (essamhassan) [reply core/config/envvar/schema.go]: removed, originally for verbosity

**Comment by essamhassan** [comment]
> changelog to reflect the new env variable.

was thinking to do this when underrlying functionality is ready since changelog is nop visible?

**Comment by essamhassan** [comment]
> Should this overall functionality be behind a global `Feature*` switch as well? Or would that just be redundant?

I'd say this is redundant to config?

**Pull request merged**